CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Removed
+- Remove `videoSendHealthDidChange`, `videoSendBandwidthDidChange`, and `videoReceiveBandwidthDidChange`.
```

Comment:
Maybe it's better to mention why we remove this?

Reply:
Updated